### PR TITLE
Update Pantheon and Colima docs - Add step to set DNS Nameserver IP address 

### PR DESCRIPTION
**Pull request by tekNorah**
## The Problem/Issue/Bug:
[Unable to pull db from pantheon after switching to colima](https://github.com/drud/ddev/issues/4035)

## How this PR Solves The Problem:
Add steps to docs regarding setting a DNS Nameserver IP address in Colima configuration
## Automated Testing Overview:

No tests included, only docs have been updated.
## Related Issue Link(s):
[Unable to pull db from pantheon after switching to colima](https://github.com/drud/ddev/issues/4035)
## Release/Deployment notes:

No anticipated deployment steps.

<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/4076"></a>

### CODE REVIEW

File: pkg/ddevapp/dotddev_assets/providers/pantheon.yaml.example [hunk 1]
```diff
@@ -28,11 +28,13 @@
...
 #
-# 9. `ddev restart`
+# 9. If using Colima, may need to set an explicit nameserver in `~/.colima/default/colima.yaml` like `1.1.1.1`. If this configuration is changed, may also need to [restart colima](https://ddev.readthedocs.io/en/stable/users/install/docker-installation/).
 #
-# 10. Run `ddev pull pantheon`. The ddev environment  download the Pantheon database and files, and import the database and files into the ddev environment. You should now be able to access the project locally.
+# 10. `ddev restart`
 #
-# 11. Optionally use `ddev push pantheon` to push local files and database to Pantheon. Note that `ddev push` is a command that can potentially damage your production site, so this is not recommended.
+# 11. Run `ddev pull pantheon`. The ddev environment  download the Pantheon database and files, and import the database and files into the ddev environment. You should now be able to access the project locally.
```
Comment (rfay) [inline pkg/ddevapp/dotddev_assets/providers/pantheon.yaml.example:35]:
```suggestion
# 11. Run `ddev pull pantheon`. The ddev environment will download the Pantheon database and files using terminus and will import the database and files into the ddev environment. You should now be able to access the project locally.
```

**Review by rfay — CHANGES_REQUESTED** [review]
You can pull these as a batch if you're OK with them. Just "add to batch" and then make them one push. And if you're ambitious, add "nameserver" to the .spellcheckwordlist. If you're not, change the occurrences to "name server"

**Comment by rfay** [comment]
Your edited docs are at https://ddev--4076.org.readthedocs.build/en/4076/

File: docs/content/users/providers/pantheon.md [hunk 2]
```diff
@@ -31,8 +31,10 @@ If you have ddev installed, and have an active Pantheon account with an active s
 
 8. In your **project's** .ddev/providers directory, copy pantheon.yaml.example to pantheon.yaml (_Note that this refers to your project .ddev folder and not the global .ddev folder_).  Edit the `project` environment variable under `environment_variables`. It will be in the format `<projectname>.<environment>`, for example `yourprojectname.dev` or (in cases of ambiguity) `<project_uuid>.<environment>`, for example `009a2cda-2c22-4eee-8f9d-96f017321555.dev`.  
 
-9. `ddev restart`
+9. If using Colima, may need to set an explicit nameserver in `~/.colima/default/colima.yaml` like `1.1.1.1`. If this configuration is changed, may also need to [restart colima](https://ddev.readthedocs.io/en/stable/users/install/docker-installation/).
```
Comment (tekNorah) [inline docs/content/users/providers/pantheon.md:34]: @rfay - I notice the "restart Colima" link does not actually link directly to the macOS: Colima tab.  See: https://ddev--4076.org.readthedocs.build/en/4076/users/providers/pantheon/

Any suggestions here?

Comment (tekNorah) [reply docs/content/users/providers/pantheon.md:34]: Strangely, I see this works: https://ddev--4076.org.readthedocs.build/en/4076/users/install/docker-installation/#colima
But, this: https://ddev.readthedocs.io/en/stable/users/install/docker-installation/#colima

Does not?

Comment (rfay) [reply docs/content/users/providers/pantheon.md:34]: It should be a relative link in the docs, not a link to ddev.readthedocs.io, thanks for catching that.

Comment (rfay) [reply docs/content/users/providers/pantheon.md:34]: OTOH, I'm not sure a link is needed at all. But if a link, it should be a relative link.

Comment (tekNorah) [reply docs/content/users/providers/pantheon.md:34]: ok, I'll just remove the link

**Pull request merged**